company-profiles/kiprosh.md

```diff
@@ -2,47 +2,43 @@
 
 ## Company blurb
 
-[Kiprosh](http://kiprosh.com) is software product development firm building powerful Web & Mobile applications for everyone from multinational corporations to startups and consultancy companies. We specialise in Ruby on Rails, React.js, Ember.js, iOS, Android, .NET and Agile way of application development.
+[Kiprosh](https://kiprosh.com) is a software product development firm crafting high impact high performance scalable web & mobile applications. We are headquartered in Mumbai, India with an office in Chicago, USA.
 
-Our happy clients are our best credentials. Our clients are consultancy firms, non-profit organisation, small to medium businesses and startups like [Realvolve](http://realvolve.com/), [CrossRoads](http://crossroads.org.hk/), [GoGoVan](http://gogovan.com.hk/en/), [Thought-Sauce](http://thought-sauce.com/), [Rayku](http://rayku.com/home) and [Renatus](http://www.myrenatus.com/). Here is our [Client List](http://kiprosh.com/clients) with details of our work.
+Our happy clients are our best credentials. Our international clients are consultancy firms, non-profits, medium to large size businesses like Crossroads, Renatus, All American Entertainment, EParamus, Realvolve, GoGoVan, Thought-Sauce, 3Qi-Labs, Rayku and many more. Here is our client list with details of our work.
```

Comment:
This text says "Here is our client list with details of our work" but the link to the client list has been removed in this PR.

Reply:
Thanks @nylen Good catch. 👍